### PR TITLE
Search filter on main list.

### PR DESCRIPTION
**Pull request by tenfef**
I added some basic search filtering on the main list.
Made the navigation item change based on currently selected filter

### CODE REVIEW

File: AfrikaBurn/BurnElementsViewController.swift
Comment (danielgalasko) [inline AfrikaBurn/BurnElementsViewController.swift]: this could be easier done using a guard statement:

```swift
guard let searchText = searchText, searchText.isEmpty == false else {
    return
}
```

File: AfrikaBurn/BurnElementsViewController.swift
Comment (danielgalasko) [inline AfrikaBurn/BurnElementsViewController.swift]: In swift you dont need to use `self` :)

File: AfrikaBurn/BurnElementsViewController.swift
Comment (danielgalasko) [inline AfrikaBurn/BurnElementsViewController.swift]: I guess this does make sense 😄

File: AfrikaBurn/BurnElementsViewController.swift
Comment (danielgalasko) [inline AfrikaBurn/BurnElementsViewController.swift]: Wonder if we should rather search the longBlurb?

File: .gitignore
Comment (danielgalasko) [inline .gitignore]: what kind of resources are you ignoring here?

**Review by danielgalasko — CHANGES_REQUESTED** [review]
This PR will be heavily affected by #4 not sure if you have taken a look at that but it does significantly change how this controller works. So maybe lets get some consensus there on filtering and then continue?

Otherwise I definitely think we can start by showing the elements as annotations to the main map view before we add search. thoughts?

File: .gitignore
Comment (tenfef) [reply .gitignore]: i had all my icon and splashscreen exports here, before i dragged them into the xcode assets. I like to keep them separate just in case i need them in future.

Comment (danielgalasko) [reply .gitignore]: maybe put them somewhere else on your machine or an your personal gitignore? not sure we would want something like this in our gitignore file otherwise it gets messy :)

**Comment by tenfef** [comment]
Have cleaned up the requested changes. Will wait until #4 is merged in and then I will fix up if needed.

**Comment by tenfef** [comment]
Closing as has been moved to PR #10

**Pull request closed without merging**
